src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -573,7 +573,7 @@ private IMethodSymbol Instantiate(IMethodSymbol method, IList<ITypeSymbol> invoc
                 }
 
                 var typeArguments = method.ConstructedFrom.TypeParameters.Select(tp => bestMap.GetValueOrDefault(tp) ?? tp).ToArray();
-                return method.Construct(typeArguments);
+                return method.ConstructedFrom.Construct(typeArguments);
```

Comment:
Be sure to pull from master as this was fixed yesterday and you'll have a conflict.